src/includes/performance/uniform-sample-rate/dotnet.mdx

```diff
@@ -0,0 +1,3 @@
+To do this, set the <PlatformIdentifier name="traces-sample-rate" /> option in your `SentrySdk.Init()` to a number between 0 and 1. With this option set, every transaction created will have that percentage chance of being sent to Sentry. (So, for example, if you set <PlatformIdentifier name="traces-sample-rate" /> to `0.2`, approximately 20% of your transactions will get recorded and sent.) That looks like this:
```

Comment:
Since this will be common for .NET and ASP.NET, Xamarin, better go generic:

```suggestion
To do this, set the <PlatformIdentifier name="traces-sample-rate" /> option when you initialize the SDK to a number between 0 and 1. With this option set, every transaction created will have that percentage chance of being sent to Sentry. (So, for example, if you set <PlatformIdentifier name="traces-sample-rate" /> to `0.2`, approximately 20% of your transactions will get recorded and sent.) That looks like this:
```